rfcs/0000-rename-unstable.md

```diff
@@ -0,0 +1,127 @@
+---
+feature: rename-unstable
+start-date: 2023-06-16
+author: @K900
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Users often interpret Nixpkgs "unstable" as "contains bugs", not the intended meaning of "can have breaking changes".
+To avoid this confusion, we should rename the "unstable" branch to something that doesn't have those connotations.
+
+# Motivation
+[motivation]: #motivation
+
+When introducing people to Nix and NixOS, the very first question that often arises is "why are we using unstable?".
+This is because the word "unstable" has two somewhat overloaded meanings - end users usually think about _application stability_
+(i.e., roughly, the "WTFs per hour"), whereas developers think about _API stability_. This results in a lot of confusion,
+especially for new users who are often told to "don't worry, just run unstable", which is a pretty scary prospect given the
+initial assumption of what "unstable" really refers to.
+
+# Detailed design
+[design]: #detailed-design
+
+## Aside: on the naming of cats
+
+This RFC intentionally does _not_ propose a new name to replace "unstable". The intent of this RFC is to not focus on the exact naming,
```

Comment:
I think we should pick a name as part of the RFC. It is hard to evaluate the benefits a better name brings without knowing what the name is. Therefore we can only evaluate the downsides without evaluating the upsides. So unfortunately we have to assume that this RFC has negative value. I don't feel comfortable accepting an RFC with just a "it will be great, trust us" promise for the advantages.